### DISCUSSION TITLE
Is this bannable?

### DISCUSSION THREAD

**Discussion by ultra-black** [SudhanPlayz/Discord-MusicBot#725] [Q&A] [answered]
Seems like a stupid question and it is.
Is this bannable?

**Comment by DarrenOfficial** [comment, marked as answer]
What is "bannable", like what are you referring to here

**Reply by ultra-black** [reply]
What I'm referring to is your discord account being deleted for using the bot.
As I have said it's a stupid question.

**Reply by DarrenOfficial** [reply]
No, discord will not ban you for using the bot.

**Reply by ultra-black** [reply]
Thanks
